assignment08/index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+    include "includes/header.php";
+
+    $first_name = 'Dissa';
+    echo $first_name;
+    echo "<br>";
+    $last_name = 'Kondo';
+    echo $last_name;
+    echo "<br>";
+    $my_age = 36;
+    echo $my_age;
+    echo "<br>";
+    $my_weight = 170;
+    echo $my_weight;
+    $val = 1/2.2;
+    function multiply($num1, num2) {
+        return num1 * num2;
+    }
```

Comment:
this is a function - but not the one the assignment calls for. I would expect something that starts:

Function poundsToKG($lbs){
   //calculate weight in kg here
   //return weight in kg
}